nixos/roles/devhost/fc-devhost.py

```diff
@@ -271,6 +271,7 @@ def ensure(
                     # Download the base image. We rename the file afterwards
                     # to ensure that the image is fully there.
                     r = requests.get(image_url)
+                    r.raise_for_status()
```

Comment:
This then causes the deployment process to be aborted, right? Some retries could be desirable for a better UX, but this at lease prevents incorrect base image data from becoming persisted.

Reply:
Correct